scripts/webidl/convert.js

```diff
@@ -30,6 +37,104 @@ const transformer = new Webidl2js({
     return `
       return ${identifier}(globalObject, interfaceName, new.target);
     `;
+  },
+  // https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#reflecting-content-attributes-in-idl-attributes
+  processReflect(idl, implObj) {
+    const reflectAttr = idl.extAttrs.find(attr => attr.name === "Reflect");
+    const attrName = (reflectAttr && reflectAttr.rhs && reflectAttr.rhs.value.replace(/_/g, "-")) || idl.name.toLowerCase();
```

Comment:
As mentioned, I would like to move this to strings instead of identifiers. But, we can do that in a followup.

Reply:
Done.